examples/cluster/plot_cluster_comparison.py

```diff
@@ -3,24 +3,25 @@
 Comparing different clustering algorithms on toy datasets
 =========================================================
 
-This example shows characteristics of different
-clustering algorithms on datasets that are "interesting"
-but still in 2D. With the exception of the last dataset,
-the parameters of each of these dataset-algorithm pairs
-has been tuned to produce good clustering results. Some
-algorithms are more sensitive to parameter values than
-others.
```

Comment:
I think this is important to keep this discussion here as well since it relates to hyperparameter tuning that was done to get nice pictures.